docs/releases/1.10.txt

```diff
@@ -131,6 +131,12 @@ Minor features
   :func:`~django.contrib.auth.views.login` view allows redirecting
   authenticated users visiting the login page.
 
+* The new :class:`~django.contrib.auth.backends.AllowAllUsersModelBackend` and
+  :class:`~django.contrib.auth.backends.AllowAllUsersRemoteUserBackend` ignore
+  the value of ``is_active`` on a user instance as opposed to the existing
```

Comment:
ignore the value of User.is_active, while ModelBackend and RemoteUserBackend now reject inactive users.